devel/git-town/Portfile

```diff
@@ -0,0 +1,24 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+PortSystem          1.0
+PortGroup           golang 1.0
+
+go.setup            github.com/git-town/git-town 7.6.0 v
```

Comment:
Please add a `revision` field:
```
go.setup            github.com/git-town/git-town 7.6.0 v
revision            0
```

`revision` fields are incremented when making a change to a port when its version number is not changing.